login.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+
+<body>
+  <div class="login-master-container">
+    <div class="login-container">
+      <img class="wolox-logo" src="Images/LogoWolox.png" alt="wolox logo">
+      <form class="login-form">
+        <p class="text"> Email </p>
+        <input class="credentials-box">
+        </input>
```

Comment:
input type? and remember the input is self closed: `<input ... />` not `<input></input>`